src/lib/core/CHIPConfig.h

```diff
@@ -1368,3 +1364,80 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #ifndef CHIP_CONFIG_MAX_SUBSCRIPTION_RESUMPTION_STORAGE_CONCURRENT_ITERATORS
 #define CHIP_CONFIG_MAX_SUBSCRIPTION_RESUMPTION_STORAGE_CONCURRENT_ITERATORS 2
 #endif
+/**
+ * @brief Indicates the absence of a Scene table entry.
+ */
+#ifndef CHIP_CONFIG_SCENES_TABLE_NULL_INDEX
+#define CHIP_CONFIG_SCENES_TABLE_NULL_INDEX 0xFF
```

Comment:
This is an implementation detail. Maybe leave in SceneTable impl. When would this be overridden?

Reply:
Moved to implementation.